apps/custom-config/vt-word.js

```diff
@@ -53,11 +54,11 @@ class VtWord extends BaseConfig {
       if (realQueryObj.action === SWITCH_VT_UPDATE) {
         logger.info(`turen update ${realQueryObj.txt}`)
         this.activity.turen.deleteVtWord(realQueryObj.oldTxt)
-        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py)
+        this.activity.turen.addVtWord(realQueryObj.txt, realQueryObj.py, realQueryObj.margin_index, realQueryObj.cloud_confirm)
```

Comment:
For compatibility, we should check the new fields `margin_index` and `cloud_confirm`.

Reply:
has confirm with font-end turen engineer , it's ok use this api for compatibility with other vt-word setting vertion

Reply:
It's not about turen, if `margin_index` is undefined, `caps.writeDouble(margin_index)` may generate an error.

Reply:
yes, has add assert value check function on vt-word file